### DISCUSSION TITLE
fastlane it is possible to send the first Android and IOS publication

### DISCUSSION THREAD

**Discussion by ederluca** [fastlane/fastlane#16832] [Q&A] [answered]
good afternoon

First I come to thank the developers immensely for the fastlane tool, but I have a question currently I send the update for android apps and ios goes fastlane, but my question is the following is possible to send the first publication via fastlane currently I perform the process manually on stores of both Google and Apple is it possible to automate this shipping too?

Currently I perform screen capture, icon generation and export of apk and ipa files to later send manually to stores.

**Comment by crazymanish** [comment, marked as answer]
Hello @ederluca 
Good afternoon, 

I am sorry, failed to understand what you mean by **_first publication_**?
Are you looking for [**produce**](https://docs.fastlane.tools/actions/produce/) tool? 🤔 

Thanks!

**Reply by ederluca** [reply]
Let me explain the first publication I am reporting would be the creation of the app in stores, that is, it would be the creation of the app.

**Reply by crazymanish** [reply]
Yup, then you need to use **Fastlane produce** tool for iOS :)
https://docs.fastlane.tools/actions/produce/

I am iOS developer, so not sure about Android tool...request you to wait for someone from Android developer or @joshdholtz :p

**Reply by crazymanish** [reply]
This is an example to create a brand new app in Apple's AppStoreConnect, Enjoy! 🚀
https://docs.fastlane.tools/actions/produce/#2-examples

**Reply by ederluca** [reply]
Thank you I will study this issue already on ios waiting for a help for android.

**Reply by joshdholtz** [reply]
@ederluca It is possible with iOS as mentioned above with the `produce`. However, it is not possible with Android. The Google API SDK does not give us that functionality 🤷‍♂️

**Reply by ederluca** [reply]
Thanks for the clarification it will help me a lot having the option for ios
